core/integration/kafka/protos/kafka.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
@awegrzyn let me know if the format & data suits you

Reply:
Looks good to me, just 1 comment, I think it might be handy if information whether this is `SOR`/`EOR`... could be also included in the message. Eg. when you subscribe to multiple topics it's way easier to read it from field rather than looking from which topic it comes from.

Reply:
I am afraid we don't have access to the last transition from within "enter_state" callback, thus I do not know myself if the FSM reaches e.g. CONFIGURED from CONFIGURE or STOP.
Perhaps @teo could confirm. Maybe we could store the last transition in `varStack`?

Reply:
I was referring to `state` variable that you also use to select the topic.

Reply:
There is `state` in `EnvInfo`.